--- gnome-alsamixer-0.9.6/src/Makefile.am.orig	2003-09-01 14:34:41.000000000 +0200
+++ gnome-alsamixer-0.9.6/src/Makefile.am	2005-05-14 14:26:48.000000000 +0200
@@ -4,10 +4,6 @@
 	-DPACKAGE_DATA_DIR=\""$(datadir)/gnome-alsamixer"\"        \
 	-DPIXMAP_ICONDIR=\""$(datadir)/pixmaps/gnome-alsamixer"\"  \
 	-DPACKAGE_LOCALE_DIR=\""$(prefix)/$(DATADIRNAME)/locale"\" \
-	-DG_DISABLE_DEPRECATED					   \
-	-DGDK_DISABLE_DEPRECATED				   \
-	-DGTK_DISABLE_DEPRECATED				   \
-	-DGNOME_DISABLE_DEPRECATED                                 \
 	@PACKAGE_CFLAGS@
 
 bin_PROGRAMS = gnome-alsamixer
